src/Types.jl

```diff
@@ -1354,20 +1360,34 @@ end
 Base.@kwdef struct PackageInfo
     name::String
     version::Union{Nothing,VersionNumber}
+    tree_hash::Union{Nothing,String}
     ispinned::Bool
     isdeveloped::Bool
+    is_tracking_registry::Bool
```

Comment:
What does this one do?

Reply:
it means the package is not tracking a path or tracking a repo

Reply:
perhaps there is a better name?

Reply:
No because we don't set a `git_source` when tracking by path. But it is redundant because it is equivalent to `!pkg.isdeveloped && git_source === nothing`